app/styles/components/advertisement-tool.scss

```diff
@@ -1,5 +1,5 @@
 // CSS3 code for animation of advertisements
-$amount-of-advertisements: 9;
+$amount-of-advertisements: 10;
```

Comment:
You added 3 new advertisements right? So then it should be 12

Reply:
Ah i now see that it is 1 advertisement containing of multiple sub-adverstiements